.ci/kitchen-macosxhighsierra-py3

```diff
@@ -9,7 +9,6 @@ runTestSuite(
     golden_images_branch: 'master',
     jenkins_slave_label: 'kitchen-slave-mac',
     kitchen_platforms_file: '/var/jenkins/workspace/pre-golden-nox-platforms.yml',
-    kitchen_verifier_file: '/var/jenkins/workspace/nox-verifier.yml',
```

Comment:
are we using a different verifier file? Im sure this is just my misunderstanding with the pytest changes?

Reply:
That's actually the default value, hence removing it.